pkg/result/filter.go

```diff
@@ -75,12 +75,36 @@ func FilterResult(ctx context.Context, result *types.Result, ignoreConf IgnoreCo
 	sort.Sort(types.BySeverity(filteredVulns))
 
 	result.Vulnerabilities = filteredVulns
+	result.MisconfSummary = calcMisconfSummary(result, misconfSummary, filteredMisconfs)
 	result.Misconfigurations = filteredMisconfs
-	result.MisconfSummary = misconfSummary
 
 	return nil
 }
 
+func calcMisconfSummary(result *types.Result, misconfSummary *types.MisconfSummary, filteredMisconfs []types.DetectedMisconfiguration) *types.MisconfSummary {
+	if misconfSummary == nil {
+		return nil
+	}
+
+	var failed, passed int
+	for _, rm := range result.Misconfigurations {
+		switch rm.Status {
+		case types.StatusPassed:
+			passed++
+		case types.StatusFailure:
+			failed++
+		}
+	}
+
+	if len(result.Misconfigurations)-len(filteredMisconfs) > 0 {
+		misconfSummary.Exceptions = len(result.Misconfigurations) - len(filteredMisconfs) - passed
```

Comment:
Misconfiguration results are also filtered elsewhere, should we take them into account? https://github.com/aquasecurity/trivy/blob/e40e1d6412ccc7be028f600e7818653949e898ff/pkg/result/filter.go#L173